CMakeLists.txt

```diff
@@ -353,91 +356,112 @@ if (OPENAL)
+	target_link_libraries(${EDITOR_EXE_NAME} -lSDL2_ttf)
   if(STEAMWORKS_ENABLED)
     #target_link_libraries(editor -lsteamworks_cwrapper -lstdc++)
-    target_link_libraries(editor -lsteam_api)
+    target_link_libraries(${EDITOR_EXE_NAME} -lsteam_api)
   endif()
 else()
   if("${CMAKE_SIZEOF_VOID_P}" EQUAL "8")
     # 64 bit
-    target_link_libraries(editor -L/usr/lib64 -lstdc++)
+    target_link_libraries(${EDITOR_EXE_NAME} -L/usr/lib64 -lstdc++)
   endif()
-  target_link_libraries(editor ${SDL2_LIBRARIES} ${SDL2_LIBRARY} ${SDL2IMAGE_LIBRARIES} ${SDL2_IMAGE_LIBRARIES} ${SDL2_NET_LIBRARIES} ${SDL2_TTF_LIBRARIES} ${SDL2TTF_LIBRARY})
+  target_link_libraries(${EDITOR_EXE_NAME} ${SDL2_LIBRARIES} ${SDL2_LIBRARY} ${SDL2IMAGE_LIBRARIES} ${SDL2_IMAGE_LIBRARIES} ${SDL2_NET_LIBRARIES} ${SDL2_TTF_LIBRARIES} ${SDL2TTF_LIBRARY})
   if (STEAMWORKS_ENABLED)
-    target_link_libraries(editor ${STEAMWORKS_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${STEAMWORKS_LIBRARY})
     #target_link_libraries(editor ${STEAMWORKS_CWRAPPER_LIBRARIES} -lstdc++)
   endif()
 endif()
-target_link_libraries(editor ${OPENGL_LIBRARIES})
-target_link_libraries(editor ${THREADS_LIBRARIES})
-target_link_libraries(editor -lm)
+target_link_libraries(${EDITOR_EXE_NAME} ${OPENGL_LIBRARIES})
+target_link_libraries(${EDITOR_EXE_NAME} ${THREADS_LIBRARIES})
+target_link_libraries(${EDITOR_EXE_NAME} -lm)
 # We need to link to Winsock if we're on Windows
 if( NOT WIN32 AND NOT APPLE)
   #Remember, Windows and Mac aren't using find_package for FMOD and PNG.
-  target_link_libraries(editor ${PNG_LIBRARY})
+  target_link_libraries(${EDITOR_EXE_NAME} ${PNG_LIBRARY})
   if (FMOD_FOUND)
-    target_link_libraries(editor ${FMOD_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${FMOD_LIBRARY})
   endif()
 endif()
 if(WIN32)
-  target_link_libraries(editor wsock32 ws2_32)
-  target_link_libraries(editor -lfmodex)
+	target_link_libraries(${EDITOR_EXE_NAME} wsock32 ws2_32)
+	target_link_libraries(${EDITOR_EXE_NAME} -lfmodex)
 endif()
 if (APPLE)
   if (FMOD_ENABLED)
-    target_link_libraries(editor -lfmodex) #Finally manually link fmod for mac.
+	  target_link_libraries(${EDITOR_EXE_NAME} -lfmodex) #Finally manually link fmod for mac.
   endif()
 endif()
-target_link_libraries(editor ${EXTRA_LIBS}) #Apple needs this for OpenGL to work.
+target_link_libraries(${EDITOR_EXE_NAME} ${EXTRA_LIBS}) #Apple needs this for OpenGL to work.
 if (OPENAL)
-  target_link_libraries(editor ${OPENAL_LIBRARY})
+	target_link_libraries(${EDITOR_EXE_NAME} ${OPENAL_LIBRARY})
   if(TREMOR_ENABLED)
-    target_link_libraries(editor ${TREMOR_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${TREMOR_LIBRARY})
   else()
-    target_link_libraries(editor ${VORBISFILE_LIBRARY})
+	  target_link_libraries(${EDITOR_EXE_NAME} ${VORBISFILE_LIBRARY})
   endif()
 endif()
 
-#install(TARGETS editor
-#  RUNTIME DESTINATION bin COMPONENT Runtime
-#)
+if (NOT APPLE AND UNIX)
```

Comment:
Any particular reason to exclude apple here? This could be useful for package managers, and the existence of an install target doesn't mean a developer has to use it if they'd rather not.